reference/Uno.Extensions.Commerce/Commerce/ViewModels/ShellViewModel.cs

```diff
@@ -36,7 +36,7 @@ public async Task Start(Route? initialRoute = null)
 			// TODO: Check for empty route
 			//&& !initialRoute.IsEmpty())
 			{
-				var initialResponse = await _navigator.NavigateRouteForResultAsync<object>(this, initialRoute);
+				var initialResponse = await _navigator.NavigateRouteForResultAsync<object>(this, initialRoute.ToString());
```

Comment:
@nickrandolph, is that expected?

Reply:
@jeromelaban , I did the parse because the new version has the override on ToString on Route.cs

![image](https://user-images.githubusercontent.com/5515535/197284265-3870e9ac-2afc-490c-a0e4-05b650613677.png)

And NavigateRouteAsync receives the route as a string:

![image](https://user-images.githubusercontent.com/5515535/197284390-b7bf51f6-655e-4b87-9f45-8cf366cf2cc5.png)

![image](https://user-images.githubusercontent.com/5515535/197284555-b37eacc1-9e39-4dd3-b31e-5acac484dc23.png)

Reply:
I think we should leave as is and make a separate issue to fix the deep linking in the commerce sample (ie the initial route).

Reply:
@nickrandolph leave as is, do you mean the change @aesteves900 made is the right one? If so, we can merge it then.

Reply:
yes change is fine